Maddori.Apple/Maddori.Apple/Global/UIComponent/BackButton.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  BackButton.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+final class BackButton: UIButton {
+    
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configUI()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        configUI()
+    }
```

Comment:
p3;

요부분에 대해서 생각이 많은데, 어떤식이 좋을진 잘 모르겠습니다
현재는 `fatalError("init(coder:) has not been implemented")`를 사용하고 있는데 앱을 강제종료 시키는 코드라 썩 내키진 않는 코드라 생각이 들어요. 하지만 스토리보드가 없어서 절대 실행되지 않기 때문에 그냥 `fatalError`를 사용하는데 왠지모를 찝찝함이 항상 있었습니다. 만약 저희가 swiftLint를 사용했더라면 항상 빨간에러를 만났을 코드입니다. 강제성이 강하기 때문에 지양하는것이 좋다고 생각하는데, 앞서 말했다시피 절대 불리지 않기때문에 또 애매한... 그런 느낌입니다..

어떤게 좋은 방법인지는 모르겠지만 현재 생각해본 대안으론 `nil`을 주는 것이었습니다.
맥락은 비슷하지만 `fatalError`를 사용하지 않는다는 이점이 있다고 생각합니다 !
```suggestion
    required init?(coder: NSCoder) { nil }
```
다른 분들의 의견이 궁금합니다 !
@dangsal @Guel-git @hwiwonK @seongmin221

Reply:
저도 fatalError 에 대해서 좀 끔끔 하고 있었는데 일단 nil 방법이 가장 좋을거같다? 라고 생각이 듭니다!

Reply:
흠 맞아요 저도 fatalError를 부르는건 조금 무리인 부분이라고 생각이 들어서 nil이 좋다고 생각합니다!

Reply:
오 fatalError 에 대해서 잘 몰랐는데 앱을 강제 종료하는 거라면 사용을 지양하는 게 좋을 것 같네요 ! nil로 수정해서 반영해두겠습니다 !

Reply:
[0258ede](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/0258ede9052a762faff999a6bc6d04c00a59695e)
일단 BackButton, ExitButton에만 수정해서 반영해두었습니다 ! 이외 내용들은 이 브랜치 안에서, 제가 작업한 게 아니라서 추후 한 번에 훑어볼 때 수정하는 게 좋을 것 같아요 !